agent/util-scripts/gold/pbench-clear-tools/test-68.txt

```diff
@@ -1,5 +1,6 @@
 +++ Running test-68 pbench-clear-tools --name=vmstat --remotes=fubar5.example.com,fubar6.example.com
 Removed "vmstat" from host, "fubar5.example.com", in tools group, "default"
+Tool /var/tmp/pbench-test-utils/pbench/tools-v1-default/fubar6.example.com/vmstat not registered for remote fubar6.example.com
```

Comment:
This appears to be a new message, can we remove it?

Reply:
Not right now its an artifact of using python library